ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -0,0 +1,155 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library    Collections
+Resource    ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup    Suite Setup
+Suite Teardown    Suite Teardown
+
+
+*** Variables ***
+${RHOAI_OPERATOR_NS}    redhat-ods-operator
```

Comment:
There is a global variable for this ${OPERATOR_NAMESPACE}, which will be automatically  updated for RHOAI and ODH with the correct namespace

Reply:
Fixed.